jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java

```diff
@@ -168,6 +175,32 @@ private static Path getResource(String path) throws URISyntaxException {
     return Paths.get(Resources.getResource(path).toURI());
   }
 
+  private static Path zipUpDirectory(Path sourceRoot, Path targetZip) throws IOException {
+    try (ZipOutputStream zipOut = new ZipOutputStream(Files.newOutputStream(targetZip));
+        Stream<Path> stream = Files.walk(sourceRoot)) {
+      for (Iterator<Path> iterator = stream.iterator(); iterator.hasNext(); ) {
+        Path source = iterator.next();
+        if (source.equals(sourceRoot)) {
+          continue;
+        }
```

Comment:
`DirectoryWalker` is an option to reduce the code here (we use it elsewhere in the plugins, although it is technically internal to jib-core). It'd be something like

```
for (Path source : new DirectoryWalker(sourceRoot).filterRoot().walk()) {
  ...
}
```

Reply:
Good catch. Very nice!